File: app/src/main/java/ch/epfl/sdp/game/TimerHelper.java
package ch.epfl.sdp.game;

public class TimerHelper {
    public static String millisToTimeString(long millis) {
        int seconds = (int) (millis / 1000) % 60;

        String secondsString;
        if (seconds < 10) secondsString = "0" + seconds;
        else secondsString = String.valueOf(seconds);

        return (int) millis / 60000 + ":" + secondsString;
    }
}

File: app/src/main/java/ch/epfl/sdp/game/TimerHelper.kt
package ch.epfl.sdp.game

/**
 * Static helper for the GameTimerFragment
 */
object TimerHelper {
    /**
     * Convert number of milliseconds to a formatted string "00:00"
     * @param millis Long: a number of milliseconds
     * @return String: a nicely formatted string
     */
    @JvmStatic
    fun millisToTimeString(millis: Long): String {
        val seconds = (millis / 1000).toInt() % 60
        val secondsString: String
        secondsString = if (seconds < 10) "0$seconds" else seconds.toString()
        return (millis.toInt() / 60000).toString() + ":" + secondsString
    }
}